data/disney

```diff
@@ -88,22 +97,30 @@ disneyme.com
 disneymeetingsandevents.com
 disneymovieinsiders.com
 disneymusicpromotion.com
+disney.my.sentry.io
 disneynewseries.com
 disneynow.com
 disneypeoplesurveys.com
+disneyplus.bn5x.net
 disneyplus.com
+disneyplus.com
+disneyplus.com.ssl.sc.omtrdc.net
+disney-portal.my.onetrust.com
 disneyredirects.com
 disneysrivieraresort.com
 disneystore.com
 disneystreaming.com
+disneystreaming.com
 disneysubscription.com
 disneytickets.co.uk
 disneyturkiye.com.tr
 disneytvajobs.com
 disneyworld-go.com
 dssott.com
-go-disneyworldgo.com
+dssott.com
 go.com
+go-disneyworldgo.com
+js-agent.newrelic.com
```

Comment:
Although Disney is a customer of New Relic, I do not think it a domain of Disney.